### ISSUE TITLE
Missing dhcp options

### ISSUE THREAD

**Issue by xhenner** [pyke369/pdhcp#1]
Missing options

The UEFI bios in some of our servers asks for these options : 

```
    "128", (TFTP Server IP address)
    "129", (Call Server IP address.)
    "130", (Discrimination string.)
    "131", (Remote statistics server IP address.)
    "132", (802.1P VLAN ID.)
    "133", (802.1Q L2 Priority.)
    "134", (Diffserv Code Point.)
    "135" (HTTP Proxy for phone-specific applications.)
```

can you implement it ?

**Comment by pyke369** [comment]
Actually, I think it would be better to let the co-process use any unknown (numerical) option in the answer, with a default type of hexstring (opaque). I will implement unknown options this way.

**Comment by pyke369** [comment]
Fixed in f5ff2d2.
